frame/bounties/src/lib.rs

```diff
@@ -151,6 +158,12 @@ pub struct Bounty<AccountId, Balance, BlockNumber> {
 	bond: Balance,
 	/// The status of this bounty.
 	status: BountyStatus<AccountId, BlockNumber>,
+	/// Subbounties counter
+	subbounty_count: BountyIndex,
+	/// Sorted list of active Subbounties
+	active_subbounty: Vec<BountyIndex>,
+	/// Tracking free balance fund for subbounties
+	subbounty_free_balance: Balance,
```

Comment:
You shouldn't need this. Just check the free balance of the parent bounty every time. Because it is possible the bounty get topup (by treasury or just someone send some money into the bounty address) and you will have incorrect value here.

Reply:
As suggested, design for subbounty fund management got changed, Now each subbounty got unique account.
state variable is removed.